src/main/java/seedu/elisa/logic/parser/FastReminderDateTimeParser.java

```diff
@@ -40,6 +44,11 @@ public LocalDateTime parseDateTime(String stringDateTime) throws ParseException
         final String unit = matcher.group("unit");
         final long longQuantity = Long.valueOf(quantity).longValue();
 
+        System.out.println(quantity);
+        if (Long.valueOf(quantity) > 100) {
+            throw new FastReminderParseException(MESSAGE_BEYOND_RANGE);
+        }
```

Comment:
Since we are limiting the value to integer value now, is there still a need to make it a long value?

Reply:
Ohh makes sense, thanks!